### PR TITLE
Add basic functionality for toggling whole layers

### PR DESCRIPTION
**Pull request by ggravlingen**
This PR adds basic functionality for toggling a whole layer in the svg-file. The layer, in turn, can contain a whole set of sensors, switches or lights.

### CODE REVIEW

**Comment by ggravlingen** [comment]
Don't merge this, there is a bug when the number of buttons > 2 that hides/unhides another layer than expected..

**Comment by ggravlingen** [comment]
False alarm, I made a mistake naming the layers.

**Review by pkozul — APPROVED** [review]

**Pull request merged**
